Examples/Scripts/FreeSurferPipelineBatchNHP.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash 
+set -e
+# Requirements for this script
+#  installed versions of: FSL6.0.4 or higher , FreeSurfer (version 6.0 or higher) , gradunwarp (python code from MGH)
+#  environment: FSLDIR , FREESURFER_HOME , HCPPIPEDIR , CARET7DIR , PATH (for gradient_unwarp.py)
+
+Usage () {
+    echo "$(basename $0) --StudyFolder=<path> --Subject=<id> --Species=<species> --RunMode=<mode> [--T2wType=<type>] [--EnvironmentScript=<path>]"
+    echo ""
+    echo "Required Options:"
```

Comment:
I think all of these options are now optional, so we don't need a distinction between required vs not.

```suggestion
    echo "$(basename $0) --StudyFolder=<path> --Subject=<id> --Species=<species> --RunMode=<mode> [--T2wType=<type>] --EnvironmentScript=<path>"
    echo ""
    echo "Options:"
```